dist/src/main/dist/conf/hazelcast.xml

```diff
@@ -83,4 +82,11 @@
         <eviction max-size-policy="ENTRY_COUNT" size="10000" eviction-policy="LFU"/>
     </cache>
 
-</hazelcast>
\ No newline at end of file
+    <topic name="ReliableITopic*">
```

Comment:
That looks more nicely, indeed. Just update your PR branch then I will adapt the test config accordingly.